lib/main.dart

```diff
@@ -38,51 +38,48 @@ class MyApp extends HookConsumerWidget {
       animationNotifier.setController(animationController);
     });
 
```

Comment:
fix an issue where the router could not identify the root put is the research bar by the user